Tests/AmplitudeTests/Utilities/IdentifyInterceptorTests.swift

```diff
@@ -0,0 +1,366 @@
+import XCTest
+
+@testable import Amplitude_Swift
+
+final class IdentifyInterceptorTests: XCTestCase {
+    private static let IDENTIFY_UPLOAD_INTERVAL_SECONDS = 1.5
+
+    private var storage: FakeInMemoryStorage!
+    private var identifyStorage: FakeInMemoryStorage!
+    private var httpClient: FakeHttpClient!
+    private var interceptor: IdentifyInterceptor!
+    private var configuration: Configuration!
+    private var pipeline: EventPipeline!
+
+    override func setUp() {
+        super.setUp()
+        storage = FakeInMemoryStorage()
```

Comment:
I guess for unit test in-memory storage could be fine, would like to make sure we tested with file storage also.

Reply:
Done.